LEAF_Request_Portal/templates/view_reports.tpl

```diff
@@ -286,7 +315,9 @@ function loadSearchPrereqs() {
             buffer += '<div class="indicatorOption"><input type="checkbox" class="icheck" id="indicators_approval_history" name="indicators[approval_history]" value="approval_history" />';
             buffer += '<label class="checkable" style="width: 100px" for="indicators_approval_history"> Approval History</label></div>';
             buffer += '<div class="indicatorOption"><input type="checkbox" class="icheck" id="indicators_days_since_last_action" name="indicators[days_since_last_action]" value="days_since_last_action" />';
-            buffer += '<label class="checkable" style="width: 100px" for="indicators_days_since_last_action"> Days since last action</label></div>';
+            buffer += '<label class="checkable" style="width: 100px" for="indicators_days_since_last_action"> Days Since Last Action</label></div>';
```

Comment:
Nothing to change here.  Just something to put into everyone's brain.  Movement onto the new UI is going to mean rows like this are going to be going away.  Variables are going to be supplied to a calling api entrance point.  So we may want to start breaking those variables out of lines like this.  The ones defined by the switch operation in this case, and then calling a different function that creates the html.  Then the return on the outer function will return a json that can have both/either just the variables and/or the html block.